src/lib/functions/server.ts

```diff
@@ -157,7 +158,7 @@ export const createHandler = function (options) {
       'client-ip': [remoteAddress],
       'x-nf-client-connection-ip': [remoteAddress],
       'x-nf-account-id': [options.accountId],
-      'x-nf-site-id': [options?.siteInfo?.id ?? 'unlinked'],
+      'x-nf-site-id': [options?.siteInfo?.id ?? UNLINKED_SITE_MOCK_ID],
```

Comment:
It's important we use the same ID here as in the blobs context, because the blobs server uses `siteID` as part of its internal keys.